network/policy/policy_windows.go

```diff
@@ -3,18 +3,49 @@ package policy
 import (
 	"encoding/json"
 	"fmt"
-	"log"
+	"strings"
 
+	"github.com/Azure/azure-container-networking/log"
 	"github.com/Microsoft/hcsshim"
+	"github.com/Microsoft/hcsshim/hcn"
 )
 
+const (
+	// protocolTcp indicates tcp protocol id for portmapping
+	protocolTcp = 6
+
+	// protocolUdp indicates udp protocol id for portmapping
+	protocolUdp = 17
+
+	// CnetAddressSpace indicates constant for the key string
+	cnetAddressSpace = "cnetAddressSpace"
+)
```

Comment:
I don't think we need these tags if they're the same as the field name

Reply:
Changed the const to CnetAddressSpace ( to match with similar const declared in the network package ). Keeping this in const section to indicate this is a field in epInfoData struct